src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,28 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMUM_AGE = 18;
+    private static final int MINIMUM_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getAge() == null || user.getAge() < MINIMUM_AGE) {
+            throw new RuntimeException("Wrong age");
```

Comment:
Splitting the condition into separate `if` statements will make the code more readable and the exception messages more informative, as described in the checklist. Each condition should throw a unique exception with a clear message.